### PR TITLE
Creates a start screen

### PR DESCRIPTION
**Pull request by alvincrespo**
## Changes

- Creates a start screen for the user to start the quiz instead of the quiz starting automatically.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -4,9 +4,23 @@ import Question from './Question.js';
 import data from'./Data.js';
 
 export default class App extends Component {
-  render() {
+  constructor() {
+    super(...arguments);
+
+    this.onStartQuiz = this.onStartQuiz.bind(this);
+
+    this.state = { startQuiz: false };
```
Comment (jks8787) [inline src/App.js]: is it possible to set this up with redux and dispatch an action? so it follows the patter used by the Question Component?

**Comment by jks8787** [comment]
@alvincrespo  looks very cool - i have a comment about how you are dealing with state. please update and I will take another look

**Comment by alvincrespo** [comment]
Yeah I think this makes sense. Since the application is a single quiz, we should probably keep that unified. 

Ill make an update to follow the pattern. Thanks!

**Comment by jks8787** [comment]
looks great @alvincrespo

**Pull request merged**